report-generator/src/main/kotlin/com/teamscale/report/jacoco/FilteringAnalyzer.kt

```diff
@@ -69,11 +69,10 @@ open class FilteringAnalyzer(
 	 */
 	@Throws(IOException::class)
 	protected open fun analyzeJar(input: InputStream, location: String): Int {
-		ZipInputStream(BashFileSkippingInputStream(input)).use { zip ->
-			return generateSequence { zip.nextEntry(location) }
+		val zip = ZipInputStream(BashFileSkippingInputStream(input))
```

Comment:
Can we maybe add a test for this scenario? IIUC it might be sufficient to zip one of the existing test jar files and feed them as input in `ConverterTest`.